bobber/bobber.py

```diff
@@ -129,6 +130,9 @@ def parse_args(version: str) -> Namespace:
     commands_parent.add_argument('--bw-threads', help='Maximum number of '
                                  'threads to use for bandwidth tests',
                                  type=int)
+    commands_parent.add_argument('stg_125k_threads', metavar='--125k-threads',
+                                 help='Maximum number of threads to use for '
+                                 '125K IO size tests', type=int)
```

Comment:
Looks like I lied to you offline. This format sets the flag as a positional argument, but not an optional flag like we expect. The format for optional flags is fairly simliar:
```python
commands_parent.add_argument('--125k-threads', dest='stg_125k_threads', help=...)
```

My bad!